modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMFormValuesToFieldsConverterImpl.java

```diff
@@ -52,6 +52,14 @@ public Fields convert(
 				ddmForm.getDDMFormFields(),
 				ddmFormValues.getDDMFormFieldValuesMap(true)));
 
+		if (ddmStructure.getClassName(
+			).contains(
+				"document.library"
+			)) {
```

Comment:
Hey @icgsbr 
We must avoid method chaining: method().method();

Reply:
I think we could improve this condition with this code:
```
if (ddmStructureKey.equals(
		DLVideoConstants.DDM_STRUCTURE_KEY_DL_VIDEO_EXTERNAL_SHORTCUT)) {

			ddmFormValues.setDefaultLocale(ddmForm.getDefaultLocale());
}
```

Reply:
I notice that you are adding a "very specific" condition in this method. I started to wonder:
-  Did we discover why the problem is happening?
- With this solution, are we really fixing the cause of the issue?

Reply:
If you need help, let me know!

Reply:
Thanks @marcelabc ! I'll take a look on it!